versioneer.py

```diff
@@ -10,7 +10,7 @@
 * https://github.com/warner/python-versioneer
 * Brian Warner
 * License: Public Domain
-* Compatible With: python2.6, 2.7, 3.3, 3.4, 3.5, and pypy
+* Compatible With: python3.5+, conda, and pypy
```

Comment:
Please undo this change, since we aren't modifying version ever itself in this PR.
```suggestion
* Compatible With: python2.6, 2.7, 3.3, 3.4, 3.5, and pypy
```